IPMUinternal/PFS-IPMU-00001/startdown.rst

```diff
@@ -17,120 +26,159 @@ Start backends first, end point to external, and monitoring systems.
+* Start monitoring services
 
-  * *prometheus*, *influxdb*, *prometheus-snmp*
+  * *prometheus* : ``prometheus`` on **dlb5-vm**
+  * *influxdb* : ``influxdb`` on **dlb3-vm**
+  * *prometheus-snmp* : ``prometheus-snmp`` on **dlb5-vm**
 
 * Start internal services
 
-  * *landfill*, *stretch*, *jessie-main*
-  * ``Windows`` : *WinP1*, *WinVSdev* and CAD
+  * ``landfill``, ``stretch``, and ``jessie-main`` on **dlb3-vm**
+  * *Windows* : ``WinP1``, ``WinVSdev`` (and CAD?) on **dlb4-vm**
 
 * Check the above VM guests running
 * Check internal services up by accessing them.
 
 Full shutdown
 ======
 
-Shutdown end point to external to prevent unwanted connection first, and 
-shutdown backends. 
-See the section `Start & shutdown VM guest with virsh`_ for the virsh command to shutdown the VM guests.
-Refer the mail (subject: \"List of virt VMs\") sent to admin weekly for VM guests running on individual hosts.
+Shutdown the end-point to the external services to prevent unwanted connection first, and 
+then back-end services. 
+See `Start & shutdown VM guest with virsh` section for the `virsh` command to shutdown the VM guests.
+Refer the mail (subject: \"List of virt VMs\") sent to admin weekly for VM guests running on individual hosts just in case.
 
-* Shutdown monitoring services (e.g. ``prometheus``, ``influxdb``) not to get unwanted (or obvious) warnings from hosts shut down.
+* Shutdown monitoring services (e.g. *prometheus*, *influxdb*) not to get unwanted (or obvious) warnings from hosts shut down.
 
-  * *prometheus*, *influxdb*, *prometheus-snmp*
+  * *prometheus* : ``prometheus`` on **dlb5-vm**
+  * *influxdb* : ``influxdb`` on **dlb3-vm**
+  * *prometheus-snmp* : ``prometheus-snmp`` on **dlb5-vm**
 
-* Shutdown internal servicse (like working shell host, simulator)
+* Shutdown internal services (like working shell host, simulator)
 
-  * *lf??*, *landfill*, *nfs??*, *stretch*, *jessie-main*
+  * ``jessie-main``, ``landfill``, and ``stretch`` on **dlb3-vm**
+  * ``nfs??`` on **dlb2-vm** (disused)
+  * ``lf??`` on **dlb7-vm** (disused)
 
-* Shutdown external servers (``pfs``, ``pfspipe``, ``pfsarch``)
+* Shutdown external servers
 
-  * *pfssrv*, *pfspipe2*, *pfsarch*
+  * *pfs* : ``pfssrv`` on **dlc2-vm**
+  * *pfspipe* : ``pfspipe2`` on **dlc2-vm**
+  * *pfsarch* : ``pfsarch`` on **dlc2-vm**
 
-* Shutdown backup service (``backup`` )
+* Shutdown backup service
 
-  * *backup*
+  * *backup* : ``backup`` on **dlb5-vm**
 
 * Shutdown iSCSI storage for pfsarch via web admin panel
 
-  * On browser type IP of pfsarch server (pas-srv, see `dnsmasq<https://github.com/Subaru-PFS/ics_dnsmasq/blob/master/hosts-ipmu/srv.conf>`_), then shutdown
-  * See PO internal wiki for sign-in information
+  * On browser type IP of *pfsarch* server (`pas-srv`, see `dnsmasq<https://github.com/Subaru-PFS/ics_dnsmasq/blob/master/hosts-ipmu/srv.conf>`_), then shutdown
+  * See PO internal wiki page for sign-in information
```

Comment:
See in advance PO internal wiki page for sign-in information.
You shutdown ``pfssrv`` before this process, so it may better to note you should do so before starting to work.